rotkehlchen/tests/unit/decoders/test_balancer.py

```diff
@@ -331,3 +332,310 @@ def test_balancer_trade(ethereum_inquirer, ethereum_accounts):
+            asset=Asset('eip155:1/erc20:0x690031313d70c2545357F4487C6a3F134C434507'),
+            balance=Balance(amount=FVal(deposit_qqq_amt)),
+            location_label=user_address,
+            notes=f'Deposit {deposit_qqq_amt} QQQ to a Balancer v1 pool',
+            counterparty=CPT_BALANCER_V1,
+            address=balancer_qqq_weth_pool_token.evm_address,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=166,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_WETH,
+            balance=Balance(amount=FVal(deposit_weth_amt)),
+            location_label=user_address,
+            notes=f'Deposit {deposit_weth_amt} WETH to a Balancer v1 pool',
+            counterparty=CPT_BALANCER_V1,
+            address=balancer_qqq_weth_pool_token.evm_address,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=167,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.RECEIVE_WRAPPED,
+            asset=balancer_qqq_weth_pool_token,
+            balance=Balance(amount=FVal(receive_amt)),
+            location_label=user_address,
+            counterparty=CPT_BALANCER_V1,
+            notes=f'Receive {receive_amt} BCoW-80QQQ-20WETH from a Balancer v1 pool',
+            address=balancer_qqq_weth_pool_token.evm_address,
+            extra_data={'deposit_events_num': 2},
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x6D3B90747dbf5883bF88fF7Eb5fCC86f408b5409']])
+def test_balancer_v1_non_proxy_exit(ethereum_inquirer, ethereum_accounts):
+    tx_hash = deserialize_evm_tx_hash('0x2a1b671429d1a2c797ba0b46735f029e69a85ba514a4d1132eab6b22c7052540')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=ethereum_inquirer, tx_hash=tx_hash)
+    user_address, pool_address, timestamp, gas_str, return_amt, withdrawn_sfrx_eth_amt, withdrawn_fxs_amt = ethereum_accounts[0], string_to_evm_address('0x11F2A400de0a2FC93a32F88D8779d8199152c6a4'), TimestampMS(1730953079000), '0.001983415737154446', '224630.905650719593790621', '11.361467105948316099', '18310.050732769125107569'  # noqa: E501
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_str)),
+            location_label=user_address,
+            notes=f'Burn {gas_str} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
```

Comment:
Again same here. This seems wrong. Withdrawal (IN) event comes first in this test. Should always be OUT first, IN later